### PR TITLE
FIX: Show iOS splash screen during bundler

### PR DESCRIPTION
**Pull request by marcosrdz**
This should make sure the BW logo is displayed all the way through the Metro package bundler process. Android next

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/n33nk464xmehq35rfr2oyedtp4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/269)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/htbvvyqehni5e2ehjltvwoadma?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/272)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: ios/BlueWallet-Bridging-Header.h [hunk 1]
```diff
@@ -2,3 +2,4 @@
 //  Use this file to import your target's public headers that you would like to expose to Swift.
 //
 
+#import <React/RCTBridgeModule.h>
```
Comment (Overtorment) [inline ios/BlueWallet-Bridging-Header.h:5]: is this line commented out..?

**Review by Overtorment — APPROVED** [review]
lgtm

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

File: ios/BlueWallet-Bridging-Header.h (hunk 1 above)
Comment (marcosrdz) [reply ios/BlueWallet-Bridging-Header.h:5]: Nope. Objc import syntax

**Comment by marcosrdz** [comment]
Nope. Thats how imports work on objc

**Pull request merged**